code.user.js

```diff
@@ -3612,6 +3632,10 @@
                 for (let i = 0; i < marketList.matchingItems.length; i++) {
                     if ($('.market_select_item', $(marketList.matchingItems[i].elm)).prop('checked')) {
                         const listingid = replaceNonNumbers(marketList.matchingItems[i].values().market_listing_item_name);
+
+                        const listingUI = $(getListingFromLists(listingid).elm);
+                        listingUI.addClass('not_checked');
```

Comment:
Maybe this needs another class to clarify, so we also know the reason why it's skipped above.